vsintegration/tests/UnitTests/QuickInfoTests.fs

```diff
@@ -444,3 +427,35 @@ module Test =
     let expected = "property MyDistance.asNautical: MyDistance with get"
     Assert.AreEqual(expected, quickInfo)
     ()
+
+[<Test>]
+let ``Automation.LetBindings.InsideModule``() =
+    let code = """
+namespace FsTest
+
+module Test =
+    let fu$$nc x = ()
```

Comment:
We should really add a test for "static let" I guess

Can do that separately though